deploy/install.go

```diff
@@ -318,7 +317,8 @@ func (dp *DeployProcess) CopySidecarProxyToCwd() {
 		return
 	}
 	tgtPath := filepath.Join(wd, sidecarImageTar)
-	if err := osRename(sidecarFilePath, tgtPath); err != nil {
+	cmd := execCommand("cp", "--recursive", sidecarFilePath, tgtPath)
```

Comment:
Are you sure we can not do this in go ? Now we are not platform-independent.

Btw, you may want to copy and preserve the permissions `cp -p`

Reply:
I'm a little confused on the word "platform-independent" isn't the 'cp' command in every distribution of linux?

Reply:
Yes, but not on Windows.

I have to confess I thought go had a built-in to copy a file but looks like not...

Reply:
added the -p flag to the execCommand